Makefile

```diff
@@ -63,6 +63,26 @@ lint-go:  # lints the Go files
 lint-md:   # lints the Markdown files
 	tools$/prettier$/node_modules$/.bin$/prettier -l .
 
+release:   # creates a new release
+	# cross-compile the binaries
+	goreleaser --rm-dist
+
+	# make Windows installer
+	make --no-print-directory msi
+
+	# create GitHub release with files in alphabetical order
+	hub release create --draft --browse --message v7.4.0 \
+		-a dist/git-town_7.4.0_linux_intel_64.deb \
+		-a dist/git-town_7.4.0_linux_intel_64.rpm \
```

Comment:
I have decided to omit 32 bit builds. 32 bit is seriously outdated at this point. 32 bit computers can only use 2 GB RAM. All hardware has been 64 bit for over 10 years. Even Linux distros are giving up on maintaining 32 bit versions. Even cell phones have been 64 bit for several generations now. Given that 64 bit OS can run 32 bit apps, there is no reason to use a 32 bit OS anymore. Hence no reason for 32 bit Git Town binaries.